vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events_database_table.cc

```diff
@@ -0,0 +1,244 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Can we please add unit tests

Reply:
We've decided these will be added in a separate PR